api/application.go

```diff
@@ -652,14 +652,10 @@ func PurgeAccountStorage(context DatabaseService, account int) {
 	}
 }
 
-func FormatShaSum(hash [sha256.Size]byte) string {
-	return hex.EncodeToString(hash[:])
-}
```

Comment:
I like this function, actually. Not as obvious what's happening without it.

Reply:
Well, I liked it too at some point. :) 

But I went to write the comment for it to resolve the lint error, and realized that the comment for that function was going to be almost exactly the same comment as the function 2 lines down from it, modulo 2 words.

And the only reason `FormatShaSum` existed at all was to call it during unit testing – usually that's not encouraged. And in the test code, the actual value of the string doesn't really matter. 

Finally, it doesn't make the `Hash()` _that_ much easier to understand (since it itself is only 2 lines, with a nice comment at the top), so I gave it the heave-ho and reduced the number of functions publicly visible in the package.

Reply:
Oh – should mention that I only added Scott as reviewer since GitHub wouldn't let me add you (because you opened the PR), but if you're willing to buy off on my reasoning, feel free to merge in.